packages/paste-core/components/toggle/src/ToggleHelpText.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import type {ToggleHelpTextProps} from './types';
+import {HelpText} from '@twilio-paste/help-text';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+const ToggleHelpText = React.forwardRef<HTMLSpanElement, ToggleHelpTextProps>(
+  ({element = 'TOGGLE_HELP_TEXT', children, helpTextId, ...props}, ref) => {
+    return (
+      // @ts-ignore TODO:no 50px tokens
+      <Box {...safelySpreadBoxProps(props)} as="span" display="block" element={element} marginLeft="50px" ref={ref}>
```

Comment:
thought: I think we can remove `as="span" display="block"` because they negate each other?